x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -147,9 +147,7 @@ function useServicesFetcher() {
     servicesData: mainStatisticsData,
     servicesStatus: mainStatisticsStatus,
     comparisonData,
-    isLoading:
-      mainStatisticsStatus === FETCH_STATUS.LOADING ||
-      comparisonStatus === FETCH_STATUS.LOADING,
+    isLoading: mainStatisticsStatus === FETCH_STATUS.LOADING,
```

Comment:
Not sure if we need to return `isLoading` since `servicesStatus` is already returned. 

```suggestion
```

Reply:
Btw. any reason why `mainStatisticsData` is renamed to `servicesData`? Keeping the same name often makes it easier to read.

Reply:
I return isLoading and the status because the status is needed at [NoServicesMessage](https://github.com/elastic/kibana/pull/108256/files/70304b0f94693d34061baa41867a570c6677943c#diff-1f8752bfc4d79520010b5d507b96176f454830da1dc8e2091578f66e42007d02R197-R199) to show an error message, but it should only be render if it is not loading.

Reply:
I opened a new PR with the suggested changes https://github.com/elastic/kibana/pull/108531